### PR TITLE
PB-48 setup string internationalization

### PR DESCRIPTION
**Pull request by ChidimmaOfodum**
# (PB-48) setup string internationalization

## Related Ticket/PR
- PB-48
---

## Changes
- This PR sets up all the strings with typesafe-i18n internationalization using English (en) locale only. Other locales to be included when need be
---

## How to run
- npm start: runs the application and typesafe-i18n simultaneously
---

## Requirements Before Merging
- Merge this [PR]()

## Notes
- We can directly import from JSON files as opposed to ts, which would make integration with tools like Weblate simpler. 
https://github.com/ivanhofer/typesafe-i18n/discussions/151

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,14 +3,16 @@
 	"version": "0.0.1",
 	"private": true,
 	"scripts": {
+		"start": "npm run dev & npm run typesafe-i18n",
```
Comment (skinofstars) [inline package.json:6]: Should this now be the default way of starting the app? Or should we be adding `npm run typesafe-i18n` to the `dev` command? I'm inclined to think the latter, as that's the standard for sveltekit apps, but if uyou think the former, then you need to update the README

File: package.json [hunk 2]
```diff
@@ -22,13 +24,15 @@
 		"eslint": "^8.28.0",
 		"eslint-config-prettier": "^8.5.0",
 		"eslint-plugin-svelte3": "^4.0.0",
+		"npm-run-all": "^4.1.5",
```
Comment (skinofstars) [inline package.json:27]: Is this used anywhere?

**Comment by ChidimmaOfodum** [comment]
> Just want to note, we can directly import from JSON files as opposed to `ts`, which would make integration with tools like Weblate simpler.
> 
> [ivanhofer/typesafe-i18n#151](https://github.com/ivanhofer/typesafe-i18n/discussions/151)

> Awesome, I will make note of this in the pull request template for easy referencing.

File: package-lock.json [hunk 3]
```diff
@@ -17,82 +17,20 @@
 				"eslint": "^8.28.0",
 				"eslint-config-prettier": "^8.5.0",
 				"eslint-plugin-svelte3": "^4.0.0",
+				"npm-run-all": "^4.1.5",
```
Comment (skinofstars) [inline package-lock.json:20]: make sure you've run install again after you remove anything from package.json

**Review by skinofstars — APPROVED** [review]

**Pull request merged**